app/views/works/index.html.erb

```diff
@@ -0,0 +1,71 @@
+<h3>List of Works</h3>
+
+<h4>Albums</h4>
+
+<table>
+<tr>
+<th>Votes</th>
+<th>Title</th>
+<th>Created By</th>
+<th>Published</th>
+<th>Upvote</th>
+
+
+</tr>
+<% @works.where(category: "album").each do |work| %>
```

Comment:
it might be better form to establish `@albums = @works.where(category: "album")` in your controller action rather than place this logic in the view, and then use `@albums`